impl/sql/mutationstorage/mutation_logs_test.go

```diff
@@ -57,45 +56,6 @@ func TestLogsAdminIntegration(t *testing.T) {
 		})
 }
 
-func TestRandLog(t *testing.T) {
```

Comment:
I feel like the replacement to this test ought to be something that confirms the entry was written to the log you specified? If you reintroduced the randomness in the mutationstorage layer and ignored the specified logID, would any test fail?

Reply:
I think that case is already covered by the storage layer integration tests: 
core/integration/storagetest/mutation_logs.go